pages/glossary/web-page.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Web page
+key: web-page
```

Comment:
Same comment as I have left in other one about webpage definition.
I would prefer if we do not add a definition & later update all rules that need webpage definition based on the conclusion arrived on whether we need this.